cg/cli/clean.py

```diff
@@ -78,10 +78,11 @@ def mip(context, yes, sample_info):
 @clean.command()
 @click.argument("bundle")
 @click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.oprtion("-d", "--dry-run", is_flag=True, help="show files that would be cleaned")
 @click.pass_context
-def scout(context, bundle, yes):
+def scout(context, bundle, yes, dry_run):
     files = []
-    for tag in ["bam", "bai", "bam-index"]:
+    for tag in ["bam", "bai", "bam-index", "cram", "crai", "cram-index"]:
```

Comment:
Not sure about what additional file tags are added to hk with cram-files, but I'm guessing this should cover it?

Reply:
HS: affirms that you are correct